src/ReactLeafletMap/layers/CanvasVectorLayer/genericCanvasLayer.js

```diff
@@ -201,3 +201,5 @@ L.CanvasLayer = (L.Layer ? L.Layer : L.Class).extend({
 L.canvasLayer = function (options) {
 	return new L.CanvasLayer(options);
 };
+
+export default null;
```

Comment:
why?

Reply:
I have error ```(!) Import of non-existent export``` in LeafletCanvasLayer.js without this export . Don`t you?